src/core_seaice/shared/mpas_seaice_advection_incremental_remap.F

```diff
@@ -4275,29 +4275,23 @@ subroutine compute_gradient_2d(&
     integer :: iCell, iEdge, iEdgeOnCell, iCellNeighbor
 
     real(kind=RKIND) :: &
-         signGradient      ! = 1 or -1, depending on which direction is taken as positive at a given edge
+         signGradient,  &  ! = 1 or -1, depending on which direction is taken as positive at a given edge
+         tempGrad
 
     real(kind=RKIND), dimension(:,:), allocatable ::   &
          normalGrad,     & ! normal components of the gradient, defined on cell edges
-         globalGrad,     & ! gradient at cell center, in global x/y/z coordinates
-         globalGradRotate  ! rotated globalGrad vector
-
-    real(kind=RKIND), dimension(:), allocatable ::   &
-         zGrad             ! diagnostic only; should be much smaller than xGrad and yGrad
```

Comment:
Could you comment out the zGrad stuff instead of deleting? The person who wrote the advection code is no longer with us and I'd like to keep, even if commented out, the diagnostics he included.